specification/attributes/discount_handling.md

```diff
@@ -0,0 +1,44 @@
+
+While providers may use different terms to describe discounts, FOCUS identifies a discount as being a reduced price applied directly to a row or charge. Any price or cost reductions that are awarded after the fact are identified as a "Credit" Adjustment Type. One example might be when a provider offers a reduced rate after passing a certain threshold of usage or spend.
+
+## Attribute ID
+
+DiscountHandling
+
+## Attribute name
+
+Discount Handling
+
+## Description
+
+Indicates how to include and apply discounts to usage charges or rows. 
+
+## Requirements
+
+* All applicable discounts MUST be applied to each row they pertain to and MUST NOT be negated in a separate row.
+* All discounts applied to a row MUST apply to the entire charge. 
+  * If a discount only applies to a subset of the charge, then that portion of the charge MUST be split into a separate row. One with the discount and one without.
+  * If multiple discounts apply, there MUST be separate rows for each unique combination of applied discounts or undiscounted amount.
+  * Each discount MUST be identifiable using existing FOCUS columns. 
+    * Discounts applied to charges/rows from a commitment-based discount MUST include a CommitmentDiscountId.
+    * If a provider applies a discount that cannot be represented by a FOCUS column, they SHOULD include additional columns to identify the source of the discount.
+* Purchased discounts (e.g., commitment-based discounts) MUST specify CommitmentUtilization on the usage rows that received the reduced rates from the discount. 
+* Purchased discounts (e.g., commitment-based discounts) MUST be amortized.
+  * The BilledCost MUST be 0 for the discounted portion of the charge and the EffectiveCost MUST be the portion of the purchase cost that applies to this row.
+  * CommitmentUtilization MUST be "Used" for rows that received a reduced price from that commitment.
+  * If a commitment is not fully utilized, the provider MUST include a row that represents the unused portion of the commitment for that charge period. CommitmentUtilization MUST be "Not Used".
```

Comment:
If a commitment is not fully utilized, the provider MUST include a row that represents the unused portion of the commitment for that charge period. CommitmentUtilization MUST be "Not Used".
Should this be worded to align to other columns where we say Commitment Unused

Reply:
Good catch! Applying suggestion...
```suggestion
  * If a commitment is not fully utilized, the provider MUST include a row that represents the unused portion of the commitment for that charge period. Charge Subcategory MUST be "Commitment Unused".
```